core/services/feeds/service.go

```diff
@@ -50,6 +50,21 @@ var (
 		Help: "Metric to track job proposal requests",
 	})
 
+	promWorkflowRequests = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "feeds_workflow_requests",
```

Comment:
Why are these counters prefixed with "feeds_"? Is that currently the case for all job specs?

Reply:
all the metrics are that way; i think b/c they are specific to the `feeds_manager`, now called CLO. i wanted to keep the convention. @jkongie  is there a better name?

Reply:
I think it's fine perhaps to keep it the way it is now unless we rename the other two existing ones. Only comment would be that the other job types consolidate the metrics into one counter so it doesn't, for example, distinguish between a ccip commit job vs a streams, but I think it makes sense to separate this here as the job here differs so much from the other types.